clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +247,105 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStatistics(String cacheAlias, PageSource pageSource) {
+
+    ServerStoreImpl store = stores.get(cacheAlias);
+
+    registerStatistic(pageSource, cacheAlias, "allocatedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.allocatedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataOccupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataAllocatedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "occupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.occupiedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "dataOccupiedMemory" ,new Callable<Number>() {
+      @Override
+      public Number call() throws Exception {
+        return store.dataOccupiedMemory();
+      }
+    });
+
+    registerStatistic(pageSource, cacheAlias, "mappings" ,new Callable<Number>() {
```

Comment:
i thought "mappings" was the keyword that was previously used?  I actually don't like it so I will change it to entries.